html2text.go

```diff
@@ -302,6 +304,7 @@ func FromReader(reader io.Reader) (string, error) {
 	return FromHtmlNode(doc)
 }
 
+//FromString removes Unicode Byte Order Mark from a string
 func FromString(input string) (string, error) {
```

Comment:
Maybe it's worth to use the comments from https://github.com/jaytaylor/html2text/blob/master/html2text.go#L100?

Reply:
ok 👍